website/src/main/tut/docs/rules/DisableSyntax.md

```diff
@@ -48,4 +56,12 @@ DisableSyntax.keywords = [
 DisableSyntax.noTabs = true
 DisableSyntax.noSemicolons = true
 DisableSyntax.noXml = true
+DisableSyntax.noCovariantTypes = true
+DisableSyntax.noContravariantTypes = true
+DisableSyntax.noDefaultArgs = true
+DisableSyntax.noValInAbstract = true
+DisableSyntax.noImplicitObject = true
+DisableSyntax.noImplicitConversion = true
+DisableSyntax.noFinalVal = true
```

Comment:
It may seem obvious, but it should be explicitly mentioned in the docs that this rule will break valid code if you rely on extending a case class.